scripts/generate_dos_specialists_questions.sh

```diff
@@ -52,3 +68,6 @@ make_questions "service manager" "serviceManager"
 make_questions "technical architect" "technicalArchitect"
```

Comment:
Doesn't matter for the script since it only creates the question files, so order would only help with readability. Manifest order matters since it's the order of service questions, so I kept the order from the spreadsheet. Might be worth asking Ralph or Sarah if we want alphabetical in the manifest